pipelines/shopgate-project.smile-io.getCustomerId.v1.json

```diff
@@ -0,0 +1,82 @@
+{
+  "version": "1",
+  "pipeline": {
+    "id": "shopgate-project.smile-io.getCustomerId.v1",
+    "public": true,
```

Comment:
could be private since its only used by getCustomerDigest internally

Reply:
removed public